contrib/google.golang.org/grpc/grpc.go

```diff
@@ -24,14 +24,14 @@ import (
 )
 
 func startSpanFromContext(
-	ctx context.Context, method, operation, service string, rate float64,
+	ctx context.Context, method, operation, service string, rate float64, opts ...ddtrace.StartSpanOption,
```

Comment:
I would like to suggest two ways to go about this:
1. Remove the `service`, `method` and `rate` params and keep only `opts`.
2. Remove `opts` and add `kind`.

As we are now is a bit messy.

Reply:
Discussed elsewhere.